python-backend/tests/users/resources/test_core_user_resource.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import json, uuid, requests
+from tests.constants import TEST_MINE_GUID, DUMMY_USER_KWARGS
+from tests.constants import TEST_PARTY_PER_EMAIL_1, TEST_PARTY_PER_PHONE_EXT_1, TEST_PARTY_PER_PHONE_EXT_1
+from app.api.users.core.models.core_user import CoreUser
+from app.extensions import db
+
+
+@pytest.fixture(scope="function")
+def setup_info(test_client):
+    cu = CoreUser.create(TEST_PARTY_PER_EMAIL_1, TEST_PARTY_PER_PHONE_EXT_1,
+                         TEST_PARTY_PER_PHONE_EXT_1)
+    cu.save()
+
+    yield dict(
```

Comment:
So if I understand it correctly, this runs before each test? Seeds the data, yields so that the test can run, and when that's done running it cleans up the record, is that correct?
Also thank you for moving towards a more isolated cleaner testing pattern, i really like this approach

Reply:
correct, and this pattern is going to be made obsolete by James' work, but this is what we have been doing in the mean time.